scripts/qa_crowbarsetup.sh

```diff
@@ -3392,9 +3447,8 @@ EOF
         test $cephret -eq 0 || ret=104
     fi
 
-    if [ "$wanttempest" = "1" ]; then
-        scp $novacontroller:/var/lib/openstack-tempest-test/tempest.log .
-        scp $novacontroller:/var/lib/openstack-tempest-test/tempest.subunit.log .
+    if [[ $wanttempest ]] ; then
+        scp $novacontroller:/var/lib/openstack-tempest-test/tempest*.log .
```

Comment:
if the subunit log is there or not is not to be decided here
if you want to test for it, test it next to the subunit test

This is generic function that just collects all tempest logs.